presto-hive/src/main/java/io/prestosql/plugin/hive/s3/HiveS3Config.java

```diff
@@ -35,6 +35,7 @@
     private String s3AwsAccessKey;
     private String s3AwsSecretKey;
     private String s3Endpoint;
+    private PrestoS3StorageClass s3StorageClass = PrestoS3StorageClass.STANDARD;
```

Comment:
Can we directly import the enum ?

Reply:
@Praveen2112 Importing Enum directly might look off since PrestoS3SignerType and PrestoS3AclType are used without importing Enum directly.

Reply:
if static import removes readability, it's should not be used;

here it is OK to static import, except it would be inconsistent. Please keep as is (inconsistency decreases readability)